src/game-logic/reducers/computeStateForNextDay.js

```diff
@@ -58,11 +60,19 @@ export const computeStateForNextDay = (state, isFirstDay = false) =>
         applyLoanInterest,
         rotateNotificationLogs,
       ]
-  )
+
+  let newState = reducers
     .concat([adjustItemValues])
     .reduce((acc, fn) => fn({ ...acc }), {
       ...state,
       cowForSale: generateCow(),
       dayCount: state.dayCount + 1,
       todaysNotifications: [],
     })
+
+  if (newState.dayCount > 1 && newState.dayCount % 365 === 1) {
```

Comment:
I noticed during playtesting that I was awarded experience once I reached day 366 rather than 365. To me it makes more sense to give experience on day multiples of 365, and I think that would look something like this:

```suggestion
  if (newState.dayCount % 365 === 0) {
```

What do you think @lstebner?

Reply:
so, i went back and forth on this for awhile to be honest, i'm open to either but what I was thinking was the 365th day is the last day of the year, and this check happens _after_ dayCount is updated, so i _think_ day 366 is the 1st day of the year meaning this runs at "midnight" of the last day. i definitely spent too long going back and forth on this haha. open to whatever you prefer here!

Reply:
> the 365th day is the last day of the year, and this check happens _after_ dayCount is updated, so i _think_ day 366 is the 1st day of the year meaning this runs at "midnight" of the last day.

Logically this makes sense, but to me it _felt_ a little awkward from a UX perspective. Given that this is a game and not something "serious" like productivity software, I think it makes sense to bias towards what _feels_ right rather than what _is_ right for things like this.

Reply:
agree with what you're saying here, made the change!